controllers/managedocs_controller.go

```diff
@@ -1058,6 +1079,95 @@ func (r *ManagedOCSReconciler) reconcilePrometheus() error {
 	return nil
 }
 
+func (r *ManagedOCSReconciler) reconcilePrometheusRemoteWrite() (*promv1.RemoteWriteSpec, error) {
+	rhobsSecret := &corev1.Secret{}
+	rhobsSecret.Name = prometheusRemoteWriteSecretName
+	rhobsSecret.Namespace = r.namespace
+	if err := r.get(rhobsSecret); err != nil {
+		return nil, fmt.Errorf("unable to fetch rhobs secret: %v", err)
+	}
+	rhobsSecretData := rhobsSecret.Data
+	rhobsSecretID := string(rhobsSecretData[prometheusRemoteWriteIDKey])
```

Comment:
- I believe by default, the values will be of type string

Reply:
No the values will be of type []byte. But ill change it to not convert it to string since we are only using it for checking if the value exists.